src/lib/y2network/config_reader/sysconfig.rb

```diff
@@ -64,13 +64,19 @@ def find_interfaces
       # return [RoutingTable] an object with routes
       def load_routes
         # load /etc/sysconfig/network/routes
-        routes = SysconfigRoutesReader.new.config
+        routing_table = SysconfigRoutesReader.new.config
         # load /etc/sysconfig/network/ifroute-*
-        dev_routes = find_interfaces.map do |iface|
-          SysconfigRoutesReader.new(routes_file: "/etc/sysconfig/network/ifroute-#{iface.name}").config
+        dev_routing_tables = find_interfaces.map do |iface|
+          SysconfigRoutesReader.new(
+            routes_file: "/etc/sysconfig/network/ifroute-#{iface.name}"
+          ).config
         end
```

Comment:
strictly speaking all this is for reading one routing table from different sources, so this "_table(s)" is misleading

Reply:
I would say that `SysconfigRoutesReader#config` returns a`RoutingTable`, right? So dev_routing_tables is basically an array of tables, right?

Reply:
no. SysconfigRoutesReader returns one routing table. It is reading one routing table which is spread over multiple files. It read "routes" (a routing table entries) and puts them together into RoutingTable.

Reply:
I might be missing something, but `find_interfaces.map` will return an array, although `SysconfigRoutes` returns a single table.

Reply:
from system POV ```SysconfigRoutes``` returns a part of complete routing table. Real routing table is build from ifroute-* files + /etc/.../routes.